src/main/kotlin/com/example/noswimmingserver/global/security/SecurityConfig.kt

```diff
@@ -39,6 +40,9 @@ class SecurityConfig(
             .antMatchers(HttpMethod.PUT, "/student").authenticated()
             .antMatchers(HttpMethod.DELETE, "/student").authenticated()
 
+            // teacher
+            .antMatchers(HttpMethod.POST, "/teacher").hasAuthority(AuthorityUtil.TEACHER)
+
```

Comment:
🚫 **[ktlint]** <no-blank-lines-in-chained-method-calls> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)